include/trick/MonteCarlo.hh

```diff
@@ -449,6 +454,20 @@ namespace Trick {
          */
         bool get_enabled();
 
+        /**
+         * Sets #processing_input_for_MC.
+         *
+         * @see @ref MonteCarloEnabling "Enabling Monte Carlo"
```

Comment:
This has not been updated; need guidance from Trick team on how best to do this.